docs/index.rst

```diff
@@ -47,6 +47,7 @@ You can contribute to `pypdf on GitHub <https://github.com/py-pdf/pypdf>`_.
    :caption: API Reference
    :maxdepth: 1
 
+   modules/PdfDocCommon
```

Comment:
Do we really consider this the top-level class already?

Reply:
can you clarify?

Reply:
In the index, we currently start with the PdfReader and PdfWriter class, then with the helper classes/modules. Should we really put PdfDocCommon at the top as the "most important" class?

Reply:
Don't think so: I consider this class as an abstract one which is just here to centralized some code. This is why the help page should not describe the members and why the inherited functions are detailed in pdfreader and pdfwriter, proving duplicated text

Reply:
Then this line needs to be changed to move PdfDocCommon a bit more to the bottom.